src/devices/awox.ts

```diff
@@ -242,6 +242,19 @@ export const definitions: DefinitionWithExtend[] = [
         extend: [m.light({colorTemp: {range: [153, 454]}})],
         whiteLabel: [{vendor: "EGLO", model: "12239"}],
     },
+    {
+        zigbeeModel: ["EBF_RGB_Zm"],
+        model: "EBF_RGB_Zm",
+        vendor: "AwoX",
+        description: "LED with adjustable color temp on main ring; extra RGB strip for full colors.",
+        extend: [
+            m.deviceEndpoints({endpoints: {"1": 1, "3": 3}}),
```

Comment:
Can you try without this line and `meta: {multiEndpoint: true},`?

Reply:
Can you also try without this line (`m.deviceEndpoints({endpoints: {"1": 1, "3": 3}}),`)?

Reply:
Done